build/pluto/prometheus/exporters/blackbox.nix

```diff
@@ -110,7 +110,7 @@ in
       ])
       (mkDnsSdProbe "smtp_starttls" {
         names = [
-          "mail-test.nixos.org"
+          "nixos.org"
```

Comment:
This is kind of silly: since we aren't switching over the MX record for `nixos.org` yet, this isn't actually going to test our new mailserver (it'll just be testing improvmx).

If I had done what @mweinelt had wanted and made this point directly at `umbriel.nixos.org`, this wouldn't be an issue.

I'm sort of inclined to change this to be a static probe rather than a dns sd probe. Alternatively, we could just leave this alone, knowing we'll eventually switch the mx record and the right thing will happen then.

Reply:
Marking resolved.

I'm handling this bit of weirdness in this preliminary PR: https://github.com/NixOS/infra/pull/571/